doc/devel/triage.rst

```diff
@@ -0,0 +1,214 @@
+
+   Online discussions may be harder than it seems at first glance, in
+   particular given that a person new to open-source may have a very
+   different understanding of the process than a seasonned maintainer.
+
+   Overall, it is useful to stay positive and assume good will. `The
+   following article
+   <http://gael-varoquaux.info/programming/technical-discussions-are-hard-a-few-tips.html>`_
+   explores how to lead online discussions in the context of open source.
+
+
+Triage Team
+-----------
+
+
+If you would like to join the triage team:
+
+1. Correctly triage 2-3 issues.
+2. Ask someone with commit rights to recommend you to the triage team
+   (publicly or privately).
+3. Responsibly exercise you new power!
+
+Anyone with commit or triage rights may also nominate a user to be
+invited to join the triage team.
+
+
+
+Triaging operations for members of the core and triage teams
+-------------------------------------------------------------
+
+In addition to the above, members of the core team and the triage team
+can do the following important tasks:
+
+- Update labels for issues and PRs: see the list of
+  the `available github labels
+  <https://github.com/matplotlib/matplotlib/labels>`_.
+
+- Triage issues:
+
+  - **reproduce the issue**, if the posted code is a bug label the issue
+    with "status: confirmed bug".
+
+  - **identify regressions**, determine if the reported bug used to
+    work as expected in a recent version of Matplotlib and if so
+    milestone for the next bug-fix release and label as "Release
+    critical".
+
+  - **close usage questions** and politely point the reporter to use
+    `discourse <https://discourse.matplotlib.org>`_ or Stack Overflow
+    instead and label as "community support".
+
+  - **close duplicate issues**, after checking that they are
+    indeed duplicate. Ideally, the original submitter moves the
+    discussion to the older, duplicate issue
+
+  - **close issues that cannot be replicated**, after leaving time (at
+    least a week) to add extra information
+
+
+
+.. topic:: Closing issues: a tough call
+
+    When uncertain on whether an issue should be closed or not, it is
+    best to strive for consensus with the original poster, and possibly
+    to seek relevant expertise. However, when the issue is a usage
+    question, or when it has been considered as unclear for many years it
+    should be closed.
+
+
+A typical workflow for triaging issues
+--------------------------------------
+
+The following workflow [1]_ is a good way to approach issue triaging:
+
+#. Thank the reporter for opening an issue
+
+   The issue tracker is many people’s first interaction with the
+   matplotlib project itself, beyond just using the library. As such,
+   we want it to be a welcoming, pleasant experience.
+
+#. Is this a usage question? If so close it with a polite message.
+
+#. Is the necessary information provided?
+
+   If crucial information (the version of Python, the version of
```

Comment:
this is the template - might be worth linking to the templates here - did they fill out the template- especially as these are variable depending on issue type.